recipes/scramp/meta.yaml

```
@@ -0,0 +1,40 @@
+{% set name = "scramp" %}
+{% set version = "1.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 475aa6296deb2737b86e9df9098e8eca0f30c8ad1cc0a8adadb99ef012a5ceba
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [py<35]
```

Could it be ``noarch``?
```suggestion
  noarch: python
```

You can select python on the requirements

---

https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python

---

Yes, thanks !